docs/tutorials/how-to-browse-clusters.md

```diff
@@ -0,0 +1,72 @@
+# How to Browse Clusters
+
+
+Monokle is a fully integrated IDE for managing manifests. It provides instant access for debugging Kubernetes resources without looking into the YAML syntax. 
+
+In this tutorial, we have illustrated the steps to help you browse and debug the resources generated with kustomize and Helm. 
+
+Let’s get started. 
+
+**Step 1:** Launch Monokle and click on the Browse button to add your project folder containing K8 resources. 
+
+<em>**Note:** Please follow this [Getting Started](https://github.com/kubeshop/monokle#readme) guide to install Monokle 🚀</em>
```

Comment:
wrong link - this should just be (getting-started.md) - same goes for all below...

Reply:
Done